.github/labeler.yml

```diff
@@ -1,35 +1,52 @@
 documentation:
-- '**Doxyfile'
-- '**docpages/**'
-- '**/*.h'
-- '**/documentation.yml'
-- '**.cspell.json'
-- '**README.md'
-- '**SECURITY.md'
-- '**CONTRIBUTING.md'
-- '**CODE_OF_CONDUCT.md'
+- changed-files:
+  - any-glob-to-any-file:
+    - '**Doxyfile'
+    - '**docpages/**'
+    - '**/*.h'
+    - '**/documentation.yml'
+    - '**.cspell.json'
+    - '**README.md'
+    - '**SECURITY.md'
+    - '**CONTRIBUTING.md'
+    - '**CODE_OF_CONDUCT.md'
+
 build:
-- '**buildtools/**'
-- '**cmake/**'
-- '**library-vcpkg/**'
-- '**library/**'
-- '**win32/**'
-- '**CMakeLists.txt'
-- '**CMakeSettings.json'
-- '**settings.json'
-- '**dpp.pc.in'
+- changed-files:
+  - any-glob-to-any-file:
+    - '**buildtools/**'
+    - '**cmake/**'
+    - '**library-vcpkg/**'
+    - '**library/**'
+    - '**win32/**'
+    - '**CMakeLists.txt'
+    - '**CMakeSettings.json'
+    - '**settings.json'
+    - '**dpp.pc.in'
+
 packaging:
-- '**vcpkg/**'
-- '**makerelease.sh'
-- '**sign.sh'
-- '**Dockerfile'
+- changed-files:
+  - any-glob-to-any-file:
+    - '**vcpkg/**'
+    - '**makerelease.sh'
+    - '**sign.sh'
+    - '**Dockerfile'
+
 submodules:
-- '**.gitmodules'
-- '**doxygen-awesome-css/**' # Ideally, nobody should be touching this, but it's here just in-case.
+- changed-files:
+  - any-glob-to-any-file:
```

Comment:
this new syntax is horrible, I died a bit reading it.
how do the devs of this action sleep at night, those-kebab-case-psychopaths lol

Reply:
lol ikr